<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="MainWidget">
   <widget class="QFrame" name="MainDiv">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1021</width>
      <height>771</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(71, 71, 109);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="SideDiv">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>181</width>
       <height>771</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame
{
background-color: rgb(39, 39, 61);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="ExitButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>430</y>
        <width>105</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
color: rgb(255, 255, 255);

background-color: rgb(111, 136, 148);

}
QPushButton::hover
{
	
	background-color: rgb(0, 128, 103);
}
</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QFrame" name="Picture">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>120</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame
{
background-color: rgb(39, 39, 61);
}
QFrame
{
	
	image: url(:/icons/My Art/60111 [Converted].png);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>130</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
color: rgb(255, 255, 255);

background-color: rgb(111, 136, 148);

}
QPushButton::hover
{
	
	background-color: rgb(0, 128, 103);
}
</string>
      </property>
      <property name="text">
       <string>Multi-Columns-Searching</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="TopDiv">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>851</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(42, 147, 161);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QComboBox" name="SelectAlgo">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox
{
background-color: rgb(39, 39, 61);
color: rgb(255, 255, 255);
}
QComboBox QAbstractItemView
{
	background-color: rgb(0, 128, 103);
    color: rgb(255, 255, 255);
}</string>
      </property>
      <item>
       <property name="text">
        <string>Sort</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bubble Sort</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Selection Sort</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Insertion Sort</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bucket Sort</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MergeSort</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Quick Sort</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="ExitButton2">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>40</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
background-color: rgb(39, 39, 61);
color: rgb(255, 255, 255);
}
QPushButton:hover
{
	
	background-color: rgb(0, 128, 103);
}</string>
      </property>
      <property name="text">
       <string>Searching</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MultiLevelSorting">
      <property name="geometry">
       <rect>
        <x>365</x>
        <y>40</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
color: rgb(255, 255, 255);
background-color: rgb(39, 39, 61);
}
QPushButton::hover
{
	background-color: rgb(0, 128, 103);
}

</string>
      </property>
      <property name="text">
       <string>Mutil-Level-Sorting</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="BottomDiv">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>680</y>
       <width>851</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(42, 147, 161);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>600</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar::chunk
{
	
	background-color: rgb(255, 161, 77);
}
QProgressBar
{
text-align:center
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="WorkspaceDiv">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>841</width>
       <height>581</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QScrollBar" name="horizontalScrollBar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>560</y>
        <width>841</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QScrollBar" name="verticalScrollBar">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>0</y>
        <width>16</width>
        <height>551</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
